src/main/gulp/e2e-tests.js

```diff
@@ -1,36 +0,0 @@
-'use strict';
-
-var gulp = require('gulp');
-
-var $ = require('gulp-load-plugins')();
-
-var browserSync = require('browser-sync');
-
-// Downloads the selenium webdriver
-gulp.task('webdriver-update', $.protractor.webdriver_update);
-
-gulp.task('webdriver-standalone', $.protractor.webdriver_standalone);
-
-gulp.task('protractor-only', ['webdriver-update', 'wiredep'], function (done) {
-  var testFiles = [
-    'src/test/e2e/**/*.js'
-  ];
-
-  gulp.src(testFiles)
-    .pipe($.protractor.protractor({
-      configFile: 'src/test/protractor.conf.js',
-    }))
-    .on('error', function (err) {
-      // Make sure failed tests cause gulp to exit non-zero
-      throw err;
-    })
-    .on('end', function () {
-      // Close browser sync server
-      browserSync.exit();
-      done();
-    });
-});
-
-gulp.task('protractor', ['serve:e2e', 'protractor-only']);
```

Comment:
@vblagoje goje look here please: https://github.com/infinispan/infinispan-management-console/pull/4/files I will fix them to run with current master soon + re-issue fixed PR only with tests ;) and starting with development of others + external contributors should come up with more soon. You will know asap when we move somewhere here... don't worry.